pkg/apis/extensions/install/install.go

```diff
@@ -56,6 +54,15 @@ func init() {
 		return
 	}
 
+	registered.RegisterVersions(availableVersions...)
```

Comment:
I will suggest moving this to be the first line of init() method.
The compute externalVersions and enable them.
That, I think, makes the code more readable.